shipyard_blueprints/athena/shipyard_athena/athena.py

```diff
@@ -1,35 +1,224 @@
-from shipyard_templates import Database
 import boto3
 import sys
+import time
+from shipyard_templates import Database, ShipyardLogger, ExitCodeException
+from shipyard_athena.errors import exceptions as errs
+from typing import Dict, Optional
 
+logger = ShipyardLogger.get_logger()
 
-class AthenaClient(Database):
-    def __init__(self, user: str, pwd: str, region: str = None) -> None:
-        self.aws_access_key = user
-        self.aws_secret_key = pwd
+
+class AthenaClient:
```

Comment:
why isn't this be inherited from the abstract class?

Reply:
Because there is no upload method. I thought it would be nicer to not have an upload method with a pass defined.